src/features/offer/components/OfferPlaylistItem/OfferPlaylistItem.tsx

```diff
@@ -63,7 +63,7 @@ export const OfferPlaylistItem = ({
         width={width}
         height={height}
         analyticsFrom={analyticsFrom}
-        fromOfferId={analyticsFrom === 'offer' ? offer.id : undefined}
+        fromOfferId={analyticsFrom === 'offer' && offer ? offer.id : undefined}
```

Comment:
Je comprends pas trop cette ligne.
 Est-ce que c'est pareil que si on fait
```suggestion
fromOfferId={!!offer && analyticsFrom === 'offer'}
```

Reply:
Ah non j'ai rien dit c'est bon 😅